go.mod

```diff
@@ -3,18 +3,17 @@ module github.com/nspcc-dev/dbft
 go 1.20
 
 require (
-	github.com/nspcc-dev/rfc6979 v0.2.0
-	github.com/pkg/errors v0.8.1
-	github.com/stretchr/testify v1.7.0
+	github.com/nspcc-dev/rfc6979 v0.2.1
+	github.com/pkg/errors v0.9.1
```

Comment:
What?! How could it survive here?

Reply:
It's from https://github.com/nspcc-dev/dbft/pull/98/commits/70747b716e5e01e0ce87b58538d8c3121bc4d611#diff-554210d4f44afafaa3b5a9b5005b0607ed1dc518dd2b48c4344f72e41d8bacfdR51-R52.

Should we remove `rfc6979` dependency completely?

Reply:
Oh, it's about errors, will remove it.

Reply:
Fixed.